src/Microsoft.Data.SqlClient/tests/FunctionalTests/SslOverTdsStreamTest.cs

```diff
@@ -0,0 +1,347 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.Tests
+{
+    public static class SslOverTdsStreamTest
+    {
+        [Theory]
+        [InlineData(0),InlineData(3),InlineData(128), InlineData(2048), InlineData(8192)]
```

Comment:
Is there any specific reason we used all inline datas in the same run instead of having 5 different lines of inlinedata?
[InlineData(0)]
[InlineData(3)] and etc...?

Reply:
I just checked the duration your design was better by 2ms

Reply:
It's a test so performance wasn't really a huge concern. It's just personal preference because as far as I know there's no functional difference between mutltiple attribute sections or one. If There's a specific way that is preferred (and it isn't this one) I can change it.